tests/jballerina-unit-test/src/test/resources/test-src/error/error_test.bal

```diff
@@ -344,12 +344,12 @@ function bar2(){
     bar();
 }
 
-function testStackOverFlow() returns [runtime:CallStackElement[], string]? {
-    error? e = trap bar();
-    if (e is error){
-        return [e.stackTrace().callStack, e.message()];
-    }
-}
+//function testStackOverFlow() returns [runtime:CallStackElement[], string]? {
```

Comment:
Why this is commented out?

Reply:
Due to the `runtime` dependency in this test, we get compilation exception for this whole `.bal` file. 
The other tests are working properly. 
I'll update the PR with @MaryamZi 's suggestion and see.